src/Components/CreateImageWizard/formComponents/RepositoriesStatus.tsx

```diff
@@ -31,7 +36,7 @@ const RepositoriesStatus = ({ repoStatus, repoUrl }: RepositoryStatusProps) => {
           bodyContent={
             <>
               <Alert variant="danger" title="Invalid" isInline isPlain />
-              Cannot fetch {repoUrl}
+              Cannot fetch {Name}
```

Comment:
This would also change to `repoName`

Reply:
changed, thank you

Reply:
I removed the name from this file, solve it in a different way